backend/api/auth/BUILD

```diff
@@ -45,10 +45,19 @@ load("//tools/build:junit5.bzl", "junit5")
 load("@io_bazel_rules_docker//container:container.bzl", "container_push")
 
 container_push(
```

Comment:
To dry up the build files you could also write: 
```python
[
container_push(
    name = tag,
    format = "Docker",
    image = ":image",
    registry = "ghcr.io/airyhq/api",
    repository = "auth",
    tag = tag,
) for tag in ["latest, beta"]
]
```

Or even better: Add a macro to the `tools/build` folder :)

Reply:
but when I did this I got no such target